agenta-web/src/pages/apps/[app_id]/testsets/index.tsx

```diff
@@ -146,6 +146,14 @@ export default function Testsets() {
                     }}
                 />
             </div>
+
+            <ConfirmTestsetDeleteModal
```

Comment:
We should use `AlertPopup` component everywhere we require a confirmation prompt from the user to maintain consistency and achieve simplicity

Reply:
Alright, thank you!